generators/base/generator-base.mjs

```diff
@@ -322,21 +314,12 @@ export default class JHipsterBaseGenerator extends PrivateBase {
    * @param {string} outputPath - Path to customize.
    */
   applyOutputPathCustomizer(outputPath) {
```

Comment:
This will be removed, don't worth refactoring.
See https://github.com/jhipster/generator-jhipster/issues/19878.

Reply:
Let's do it step by step: I don't want to break anything during the refacto...

Reply:
Implemented at https://github.com/jhipster/generator-jhipster/pull/20277.
Needed to change everywhere that uses constants with configurable veritable.